packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 47.3.0 - October 24, 2019
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Enable diff command for layouts and permsets ([PR #1713](https://github.com/forcedotcom/salesforcedx-vscode/pull/1713))
+
+## Fixed
+
+#### docs
+
+- Fix Japanese user guide section (#1720) ([PR #1719](https://github.com/forcedotcom/salesforcedx-vscode/pull/1719))
```

Comment:
We can remove this one since we did the fix during last week's release as part of releasing the Japanese translations.

Reply:
@lcampos Are you certain? If I'm reading the logs correctly it was merged after 47.2.0.

Reply:
Yes, we had to do it straight to the `master` branch because we found it after the release. We then ported it back to `develop`.

Reply:
Okay, thanks, I'll remove it.

Reply:
Just an fyi, that was a real one-off scenario